core/modules/system/system.api.php

```diff
@@ -3039,15 +3039,59 @@ function hook_disable() {
 }
 
 /**
+ * Provide paths to classes and interfaces that can be autoloaded by PHP.
+ *
+ * Most classes and interfaces in Backdrop are autoloaded, meaning they do not
+ * need to be manually included with PHP's include_once() or require_once().
+ * Creating a new instance of a class will automatically include the needed
+ * file that contains that class.
+ *
+ * Use this hook to define the paths to classes and interfaces within your
+ * module.
```

Comment:
This is the same as the first paragraph / sentence of the docs. Combine / consolidate?